src/composables/useSpacefinderSupabase.ts

```diff
@@ -26,8 +26,22 @@ export default function useSpacefinderSupabase() {
     return data;
   }
 
+  function getBuildings() {
+    return client
+      .from("buildings")
+      .select("number, opening_hours")
+      .then(({ data, error }) => {
+        if (error) {
```

Comment:
reflect error in UI?

Reply:
There is a NotificationBar component but no straightforward way to pass messages to it from a composable, something for a ticket maybe? A general queue of messages to show could be useful for feedback (errors and info).